src/tasks/__tests__/processBulkUpload.int.test.ts

```diff
@@ -279,7 +282,7 @@ describe('processBulkUpload', () => {
 		expect(requests.getRequest.isDone()).toEqual(false);
 	});
 
-	it('should set the status of the upload to FAILED if the csv does not have a proposal_submitter_email field', async () => {
+	it('should have a correct failed state if the csv does not have a proposal_submitter_email field', async () => {
```

Comment:
It seemed somewhat silly to add an entirely separate test to check that the file size was populated alongisde the fail status... so I just changed the scope of the test.  I'm not sure if I'm violating some kind of testing best practice here though.

Reply:
I'm okay with broadening the test to include this! However…

1. [L225](https://github.com/PhilanthropyDataCommons/service/blob/582b298e9e9d49ed3ffef21daab922f0ab79283e/src/tasks/__tests__/processBulkUpload.int.test.ts#L225) would also need this change.
2. "should have a correct failed state" is verbose. I think it's okay to just say "should fail". (I appreciate the explicitness of the former, and the mild confusion about "fail" in a testing context. I still think it reads more clearly, and especially avoids the "positive-negative" whiplash of "correct failed state".) This suggestion applies for all the tests that are worded this way, including the (unintentional?) variations such as ["proper failed state"](https://github.com/PhilanthropyDataCommons/service/blob/582b298e9e9d49ed3ffef21daab922f0ab79283e/src/tasks/__tests__/processBulkUpload.int.test.ts#L365).